### ISSUE TITLE
Problem with composer setup

### ISSUE THREAD

**Issue by ghost** [grrr-amsterdam/wordpress-scaffold#49]
After commando composer setup I has problem, on my screen you can see this. Can you fix this or help me? 

<img width="1331" alt="Снимок экрана 2020-03-31 в 13 35 00" src="https://user-images.githubusercontent.com/18503614/78021964-707e6480-7354-11ea-9354-14dc5a530f6f.png">

Thank you

**Comment by HammenWS** [comment]
@kirilldorozhynskyi Thanks for you comment. There seems to be a bug in autoloading files in the composer script. I created a PR to fix this issue.

**Comment by schoenkaft** [comment]
@kirilldorozhynskyi should be fixed in `v2.3.3`. Running a new `create-project` should resolve it:

```sh
$ composer create-project grrr-amsterdam/wordpress-scaffold <project-name>
```
